code/modules/fishing/fish/types/weird.dm

```diff
@@ -0,0 +1,398 @@
+/obj/item/fish/starfish/chrystarfish
+	name = "chrystarfish"
+	fish_id = "chrystarfish"
+	desc = "This is what happens when a cosmostarfish sneaks into the bluespace compartment of a hyperspace engine. Very pointy and damaging - leading cause of spaceship explosions in 2554."
+	icon = 'icons/obj/aquarium/weird.dmi'
+	dedicated_in_aquarium_icon = 'icons/obj/aquarium/weird.dmi'
+	icon_state = "chrystarfish"
+	lefthand_file = 'icons/mob/inhands/fish_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/fish_righthand.dmi'
+	force = 12
+	sharpness = SHARP_POINTY
+	wound_bonus = -10
+	bare_wound_bonus = 15
+	armour_penetration = 6
+	demolition_mod = 1.2
+	throwforce = 11
+	throw_range = 8
+	throw_speed = 4
+	embed_type = /datum/embed_data/chrystarfish
+	attack_verb_continuous = list("stabs", "jabs")
+	attack_verb_simple = list("stab", "jab")
+	hitsound = SFX_DEFAULT_FISH_SLAP
+	pickup_sound = 'sound/items/handling/materials/glass_pick_up.ogg'
+	drop_sound = 'sound/items/handling/materials/glass_drop.ogg'
+	pickup_sound = SFX_FISH_PICKUP
+	sound_vary = TRUE
+	item_flags = SLOWS_WHILE_IN_HAND
+
+	sprite_width = 7
+	sprite_height = 9
+
+	average_size = 40
+	average_weight = 1500
+	food = /datum/reagent/bluespace
+	feeding_frequency = 10 MINUTES
+	health = 300 // it has 300 health why does it die instantly upon bein bit once..
```

Comment:
This happens because the damage taken is proportional to the initial health and is only reduced by increasing the weight so far. 

You could replace the FISH_WEIGHT_BITE_DIVISOR directive with a variable and set it to a lower value for this fish.